root/static/scripts/edit/MB/reltypeslist.js

```diff
@@ -6,9 +6,15 @@ $(function () {
 
     $(".toggle").click(function () {
 
-        $(this).parent().next(".reldetails").toggle();
-
-        if ($(this).parent().next(".reldetails").is(':hidden')) {
+        $(this)
+            .parent()
+            .next(".reldetails")
+            .toggle();
+
+        if ($(this)
+            .parent()
+            .next(".reldetails")
+            .is(':hidden')) {
```

Comment:
I think it'd be clearer to assign this to a variable first (`const isHidden = ...`) and check `if (isHidden)` here.